### PR TITLE
Fix 'test_debuginfo' from failing if a kernel symbol has multiple aliases

### PR DESCRIPTION
**Pull request by sandip4n**
The first symbol from `/proc/kallsyms` is read by `test_debuginfo` and the name obtained here is compared against the name obtained from the `BPF.ksym()` library call. In some architectures such as `powerpc64le`, a kernel symbol may have multiple aliases that refer to the same address. So, to avoid the test from failing, we need to compare the name returned by `BPF.ksym()` against all possible aliases for the given address in `/proc/kallsyms` and look for a match.

### CODE REVIEW

File: tests/python/test_debuginfo.py [hunk 1]
```diff
@@ -9,22 +9,38 @@
...
 
     def test_ksymname(self):
         sym = BPF.ksymname("__kmalloc")
         self.assertIsNotNone(sym)
         self.assertNotEqual(sym, 0)
 
     def test_ksym(self):
-        (addr, name) = self.grab_sym()
+        (addr, aliases) = self.grab_sym()
         sym = BPF.ksym(int(addr, 16))
```
Comment (yonghong-song) [inline tests/python/test_debuginfo.py]: This is not related to your change. The above uses "int(addr, 16)". It looks like "long(addr, 16)" may be better for addr? The reason that "int(addr, 16)" works could be due to later sign extension. The "addr" c binding has a type of unsigned long long.

lib.bcc_symcache_resolve_no_demangle.argtypes = [ct.c_void_p, ct.c_ulonglong, ct.POINTER(bcc_symbol)]

Comment (sandip4n) [reply tests/python/test_debuginfo.py]: I tried changing it to `long(addr, 16)` and testing on both `x86_64` and `ppc64le` systems. Works fine on both.

However, from what I have read [here](http://deeplearning.net/software/theano/tutorial/python-memory-management.html), integers in Python have a minimum size of 24 bytes, so the current code should be doing fine. Also, when the size of an integer exceeds `sys.maxint` (i.e. 2<sup>63</sup> - 1 for 64-bit systems), the minimum size becomes 36 bytes and can also continue to grow linearly in the logarithm of the integer represented.

Comment (yonghong-song) [reply tests/python/test_debuginfo.py]: Thanks for explanation. I thought that "int" in python is also 32bit. Obviously I am wrong. Your original patch is good then.

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

**Pull request merged**